traffic_ops/traffic_ops_golang/api_capability/api_capabilities.go

```diff
@@ -0,0 +1,110 @@
+package api_capability
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
+	"github.com/jmoiron/sqlx"
+)
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+func GetAPICapabilitiesHandler(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	results, usrErr, sysErr := getAPICapabilities(inf.Tx, inf.Params)
+
+	if usrErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, usrErr, nil)
+		return
+	}
+
+	if sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, nil, sysErr)
+		return
+	}
+
+	api.WriteResp(w, r, results)
+	return
+}
+
+func getAPICapabilities(tx *sqlx.Tx, params map[string]string) ([]tc.APICapability, error, error) {
+	selectQuery := `SELECT id, http_method, route, capability, last_updated FROM api_capability`
+	queryParamsToQueryCols := map[string]dbhelpers.WhereColumnInfo{
+		"id":          dbhelpers.WhereColumnInfo{"id", api.IsInt},
+		"capability":  dbhelpers.WhereColumnInfo{"capability", nil},
+		"httpMethod":  dbhelpers.WhereColumnInfo{"http_method", nil},
+		"route":       dbhelpers.WhereColumnInfo{"route", nil},
+		"lastUpdated": dbhelpers.WhereColumnInfo{"last_updated", nil},
+	}
+
+	where, orderBy, pagination, queryValues, errs :=
+		dbhelpers.BuildWhereAndOrderByAndPagination(params, queryParamsToQueryCols)
+
+	if len(errs) > 0 {
+		return nil, errors.New(
+			fmt.Sprintf(
```

Comment:
Instead of `errors.New(fmt.Sprintf(...))` consider just using [`fmt.Errorf`](https://godoc.org/fmt#Errorf).

Also, you're returning this raw database error to the client and printing nothing to the logs. You should be doing the opposite in this case; this is a server error, not a user error.